.github/workflows/php.yml

```diff
@@ -0,0 +1,47 @@
+name: Poser
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        container: [ "php74  ", "php80" ]
+
+    runs-on: ubuntu-latest
+    container: pugx/poser:${{ matrix.container }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Validate composer.json
+        run: composer validate
+
+      - name: Cache Composer packages
+        id: composer-cache
+        uses: actions/cache@v2
+        with:
+          path: vendor
+          key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-php-
```

Comment:
`${{ runner.os }}-php-${{ matrix.container }}-${{ hashFiles('**/composer.lock') }}`
and
`${{ runner.os }}-php-${{ matrix.container }}`
WDYT?